### PR TITLE
Fix/revert logs

### PR DESCRIPTION
**Pull request by muralisc**

### CODE REVIEW

**Comment by subhankarc** [comment]
@muralisc Did we check if this is causing any memory issue? May be we can run several test runs with PostgreSQL enabled in tests.

File: broker/lib/controllers/ServiceFabrikApiController.js [hunk 1]
```diff
@@ -465,13 +465,10 @@ class ServiceFabrikApiController extends FabrikBaseController {
           operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
           operationId: backupGuid,
         }))
-      .then(result => this.backupStore.getBackupFile(result)
```
Comment (kamath-prasad) [inline broker/lib/controllers/ServiceFabrikApiController.js]: why is this deleted ?

**Comment by subhankarc** [comment]
Closing it as these changes are made part of the PR 
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/326

File: broker/lib/controllers/ServiceFabrikApiController.js (hunk 1 above)
Comment (muralisc) [reply broker/lib/controllers/ServiceFabrikApiController.js]: its added https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/325/files/cfc8e173ee5f78f30bd39a4c6130a2084f785401#diff-b35054ba7edff4ba9736b79267cb63a2R278

Comment (kamath-prasad) [reply broker/lib/controllers/ServiceFabrikApiController.js]: yes , spoke to @subhankarc and understood the change.
